include/mbgl/renderer/renderer.hpp

```diff
@@ -108,6 +108,7 @@ class Renderer {
     const std::vector<PlacedSymbolData>& getPlacedSymbolsData() const;
 
     // Memory
+    void enableTileCache(bool);
```

Comment:
Perhaps this should (also) be called `setTileCacheEnabled(bool)` because it is a bit clearer that it actually disables the tile cache when passing `false`.

Reply:
Renamed. 👍